pex/pex.py

```diff
@@ -321,7 +321,7 @@ def execute(self):
     except SystemExit as se:
       # Print a SystemExit error message, avoiding a traceback in python3.
       # This must happen here, as sys.stderr is about to be torn down
-      if not isinstance(se.code, int):
+      if not isinstance(se.code, int) and se.code is not None:
         print(se.code, file=sys.stderr)
```

Comment:
This change LGTM.

However, w.r.t to previous change, we never want any exit code / message to be explicitly printed on stderr. Isn't it?  I couldn't realize the need of it even after reading comment re: avoiding traceback in python3.